vimrc

```diff
@@ -29,6 +29,16 @@ endif
 
 filetype plugin indent on
 
+" Softtabs, 2 spaces
+set tabstop=2
+set shiftwidth=2
+set shiftround
+set expandtab
+set softtabstop=2
```

Comment:
The Python folks (most non-Ruby folks, it seems) have a larger sts and ts. Is it reasonable to set this to 2 globally/is there a better global number to set it to?